packages/swing-store/src/swingStore.js

```diff
@@ -483,6 +477,12 @@ export function makeSwingStore(dirPath, forceReset, options = {}) {
     stopTrace();
   }
 
+  const internal = makeInternal(snapStore, transcriptStore, bundleStore);
```

Comment:
Man, `makeInternal` really wants a better name. Maybe this?
```suggestion
  const internal = makeSwingStoreKit(snapStore, transcriptStore, bundleStore);
```
Or alternatively just don't bother with the function at all:
```suggestion
  /** @type {SwingStoreInternal} */
  const internal = harden({ snapStore, transcriptStore, bundleStore });
```

Reply:
I like the latter, but I need a place to put `SwingStoreInternal` where it can be imported without causing a cycle. I can create a new file to hold just the type (no actual JS code). Should it be called `internal.d.ts`? Do I need to use a `@type {import('./internal.d.ts').SwingStoreInternal}` to reach it?

Reply:
ah, I see, just `internal.js` with an `export {};`, like in the new types.js